src/components/CartListing/CartListing.jsx

```diff
@@ -0,0 +1,44 @@
+import './CartListing.css'
+import { AiFillDelete } from "react-icons/ai";
+import { useEffect, useState } from 'react';
+import { useCart } from '../../context/cart-context';
+const CartListing = () => {
+  
+  const { cartState: {cart}, cartDispatch } = useCart();
+
+  const [total, setTotal] = useState();
+  useEffect(() => {
+    setTotal(cart.reduce((acc, curr) => {
+      return acc + Number(curr.price) * curr.qty
+    }, 0))
+  })
```

Comment:
You don't need to do this calculation in `useEffect()`
```
const total = cart.reduce((acc, curr) => acc + Number(curr.price) * curr.qty, 0)
```
The above code will work fine